apps/api/src/routers/guest.py

```diff
@@ -62,6 +65,7 @@ async def verify_guest(
     email: Annotated[EmailStr, Depends(guest_email)],
     passphrase: Annotated[str, Form()],
     guest_confirmation: Annotated[str, Cookie()],
+    return_to: str = "/portal",
```

Comment:
Thought: since there's already a default value provided from the initiation (the `guest_login`) function, I don't think we need to specify a default value again.